config/settings-env/endpoints.json

```diff
@@ -550,6 +550,20 @@
         "timeout": "600s",
         "input_query_strings": []
       },
+      {
+        "endpoint": "/v1beta/users/{user_id}/pipelines/{pipeline_id}/trigger:stream",
+        "url_pattern": "/v1beta/users/{user_id}/pipelines/{pipeline_id}/trigger:stream",
+        "method": "POST",
+        "timeout": "600s",
+        "input_query_strings": []
+      },
+      {
+        "endpoint": "/v1beta/sse",
+        "url_pattern": "/v1beta/sse",
```

Comment:
Will this endpoint be called after the trigger one? If so, will the trigger ID be passed as a header, or how would it work? If you could outline the sequence of requests and/or attach a small diagram it'd be great.